src/components/Account/Account.tsx

```diff
@@ -69,32 +49,68 @@ const Account = ({
     return label;
   };
 
-  const handleSubmit = (values: GenericFormValues) => {
+  const translateErrors = (errors?: string[]) => {
+    const formErrors: CustomerFormErrors = {};
+
+    errors?.map((error) => {
+      switch (error) {
+        case 'Invalid param email':
+          formErrors.email = 'Invalid email address!';
+          break;
+        case 'Customer email already exists':
+          formErrors.email = 'Email already exists!';
+          break;
+        case 'Please enter a valid e-mail address.':
+          formErrors.email = 'Please enter a valid e-mail address.';
```

Comment:
Maybe we should have `email` instead of `e-mail` to be consistent, just a small detail 🙂
Or have the other informative errors written as `e-mail `